runner/src/main.rs

```diff
@@ -413,6 +415,33 @@ fn run_client(name: &str, client: &Client) -> Step {
                 .flatten(),
             ),
         },
+        Target::NPM { package_directory } => Step::Multiple {
+            name: format!("Running the {} NPM client at {}", name, package_directory),
```

Comment:
I would make these shorter if possible, or the indentation makes them go quickly off screen :)